src/components/HeroHeader/DropdownMenu.tsx

```diff
@@ -0,0 +1,63 @@
+import Image from 'next/image';
+import React from 'react';
+import Button from '../Button/Button';
+import infoIcon from '../../../public/img/info-icon.svg';
+import calendarIcon from '../../../public/img/calendar-icon.svg';
+import contactIcon from '../../../public/img/contact-icon.svg';
+import { ABOUT, CONTACT, EVENTS } from '../../util/routeConstants';
+
+interface DropdownMenuProps {
+  onCloseMenu: () => void;
+}
+
+export default function DropdownMenu({ onCloseMenu }: DropdownMenuProps) {
```

Comment:
It looks like the way this dropdown menu is currently implemented makes it inaccessible, I cannot focus on it, and navigate the options using the keyboard. We should consider using something like this https://headlessui.com/react/menu instead.

Reply:
Fixed via [3191796](https://github.com/reactdeveloperske/reactdevske-website/pull/105/commits/3191796db3d25830c15064e91acd8598a1f49d6a)


PS. Loved the library. Thanks for suggesting!